h/static/scripts/annotator/guest.coffee

```diff
@@ -68,7 +70,6 @@ module.exports = class Guest extends Annotator
             this.publish(event, args)
       formatter: (annotation) =>
         formatted = {}
-        formatted.uri = @getHref()
```

Comment:
Seems fine to me. It doesn't make sense for us to keep storing this field when the rest of the model explicitly says we have multiple targets.